platform/java/spring/instrumented/src/main/resources/application.yml

```diff
@@ -4,9 +4,10 @@ sentry:
 spring:
   profiles: jdbc,jpa
   datasource:
-    url: jdbc:postgresql://${database.host}:${database.port}/${database.name}
+    url: jdbc:p6spy:postgresql://${database.host}:${database.port}/${database.name}
     username: ${database.username}
     password: ${database.password}
+    driver-class-name: com.p6spy.engine.spy.P6SpyDriver
```

Comment:
Is this driver required by `sentry-jdbc`?

Reply:
Yes, we do instrumentation using P6Spy https://p6spy.readthedocs.io/